Datadog.Trace.sln

```diff
@@ -1596,7 +1596,6 @@ Global
 		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{bc44a41f-1bed-4438-9f66-0ea5607906d5}*SharedItemsImports = 5
 		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{cb56ac5a-d2c1-40de-99d5-dcf9f44c9482}*SharedItemsImports = 5
 		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{deacde01-95fe-4777-b70a-f20a96aabea7}*SharedItemsImports = 5
-		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{ed3d82c5-021a-415a-a3d3-811ad85cd93c}*SharedItemsImports = 5
```

Comment:
Did you mean to remove this 🤔

Reply:
I am not sure what is it, on every build this line was removed.
Every test is passing, so I assume it's safe to merge?
Or better to revert?